### PR TITLE
add r-tframe

### PR DESCRIPTION
**Pull request by FernandezMathieu**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by FernandezMathieu** [comment]
Linked to https://github.com/conda-forge/staged-recipes/pull/18346

**Comment by dbast** [comment]
This PR contains lots of not required comments, please remove or re-generate with https://github.com/bgruening/conda_r_skeleton_helper

**Comment by FernandezMathieu** [comment]
@dbast done

**Comment by FernandezMathieu** [comment]
@dbast can you merge please ?

**Comment by FernandezMathieu** [comment]
@conda-forge/core can someone please merge this PR

**Comment by FernandezMathieu** [comment]
@dbast please can you do something ?

File: recipes/r-tframe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    See the Guide distributed as a vignette, or ?tframe.Intro for more details. (User
+    utilities are in package tfplot.)
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+    - LeoDtrt
```
Comment (dbast) [inline recipes/r-tframe/meta.yaml:52]: @LeoDtrt Do you agree being maintainer of the recipe?

**Pull request merged**